### PR TITLE
hide name check result container when name is too short

### PR DESCRIPTION
**Pull request by bolyachevets**
*Issue #: [/bcgov/entity/16998](https://github.com/bcgov/entity/issues/16998)

*Description of changes: 

Screenshot 2023-08-23 at 4 59 31 PM

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the namerequest license (Apache 2.0).

### CODE REVIEW

**Comment by pwei1018** [comment]
Temporary Url for review: https://namerequest-dev--pr-657-8w8uwoxl.web.app

**Comment by severinbeauvais** [comment]
Could this fix be applied to the [Way of Navigating feature branch](https://github.com/bcgov/namerequest/tree/feature-way-of-navigating) instead? (Feature branch is planned for deployment in October.)

The feature branch is in active development at the moment and any changes to main branch may conflict, or at least will require some careful rebasing/refactoring. Therefore, I'd prefer to merge this PR into the feature branch if possible.

File: src/components/new-request/name-check/name-check.vue [hunk 1]
```diff
@@ -60,9 +60,9 @@
           {{ nameCheckVerdict }}
         </span>
       </v-row>
-      <v-container v-if="hasIssuesConflictAlert || hasIssuesConflictCaution ||
+      <v-container v-if="(hasIssuesConflictAlert || hasIssuesConflictCaution ||
                          hasIssuesStructureAlert || hasIssuesStructureCaution ||
-                         loadingStructureCheck"
+                         loadingStructureCheck) && getShowNameCheckResult"
```
Comment (severinbeauvais) [inline src/components/new-request/name-check/name-check.vue:65]: Why not simply use `showNameCheckResult` here and remove the getter (computed)?

**Review by severinbeauvais — COMMENTED** [review]
From what I know, this won't conflict with changes in the feature branch made up to now.

File: src/components/new-request/name-check/name-check.vue (hunk 1 above)
Comment (severinbeauvais) [reply src/components/new-request/name-check/name-check.vue]: A better solution was implemented!

File: src/components/new-request/name-check/name-check.vue [hunk 2]
```diff
@@ -675,6 +680,11 @@ export default class NameCheck extends Vue {
   }
   checkAgain () {
     this.startAnalyzeName(null)
+    if (this.getErrors.includes('length')) {
+      this.showNameCheckResult = false
+    } else {
+      this.showNameCheckResult = true
+    }
```
[the remark this answers is not in the document]
Comment (severinbeauvais) [reply src/components/new-request/name-check/name-check.vue]: Looks good!

**Review by severinbeauvais — APPROVED** [review]
LGTM but please update the app version!

**Comment by pwei1018** [comment]
Temporary Url for review: https://namerequest-dev--pr-657-8w8uwoxl.web.app

**Review by ozamani9gh — COMMENTED** [review]
great

**Pull request merged**